### ISSUE TITLE
Update troubleshooting table

### ISSUE THREAD

**Issue by clearab** [MicrosoftDocs/bot-docs#1738]
PVA has updated the source for the troubleshooting table. Can you please take a look at this: https://docs.microsoft.com/en-us/power-virtual-agents/configuration-add-skills#troubleshooting-errors-during-skill-registration and merge the new information into our table? An issue should then be raised on the PVA version of the article to remove their version.

[6/18] The ABS doc update is complete and should be live within a week or so. @clearab , please coordinate with the PVA team to update their docs.

#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 489b8c9c-8bb9-9b7d-62e0-7e2853f6a3a6
* Version Independent ID: 1a60d0d1-3421-eba4-9276-e5b924f8ed6b
* Content: [Implement a skill for Power Virtual Agents - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/skill-pva?view=azure-bot-service-4.0#validation-performed-during-registering-a-skill)
* Content Source: [articles/v4sdk/skill-pva.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/v4sdk/skill-pva.md)
* Service: **bot-service**
* GitHub Login: @clearab
* Microsoft Alias: **kamrani**

**Comment by clearab** [comment]
@JonathanFingold could you please take a look at this one?

**Comment by JonathanFingold** [comment]
@clearab Mind if I add the PVA error codes to the table in the bot doc?

**Comment by clearab** [comment]
Yes please!

**Comment by JonathanFingold** [comment]
Removing these two steps, as they are not reflected in the new table:

| Validation step | Error code | Error message | Description or mitigation
| :-- | :-- | :-- | :--
| Validate skill action language || Currently, skills are only supported in English.| The skill has actions with unsupported locales. We only support skills with actions is English (`en-` locales). |
| Validate AAD app setting || The skill must be registered multi-tenant.| Verify that your AAD app is marked as multi-tenant. See [Convert app to be multi-tenant](/azure/active-directory/develop/howto-convert-app-to-be-multi-tenant#update-registration-to-be-multi-tenant) |

**Comment by JonathanFingold** [comment]
Once the PR is merged into master, it should go live within a week.
